src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -208,106 +221,167 @@ contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCou
         view
         returns (bytes32)
     {
-        return keccak256(abi.encodePacked(_currentAddress, _newAddress, rotationNonce));
+        return keccak256(abi.encodePacked(_currentAddress, _newAddress, updateNonce));
     }
 
-    /// @notice Verify signature over data
-    /// @param data Signed data
-    /// @param signature over data
-    /// @param account signer
-    /// @return true if signature is valid and from account
-    function _verify(bytes32 data, bytes memory signature, address account)
-        public
-        pure
-        returns (bool)
+    /// @notice Add new security council to be included in security council management system. Only DAO can call.
+    /// @param _securityCouncilData data for security council to be added
+    function addSecurityCouncil(SecurityCouncilData memory _securityCouncilData)
```

Comment:
Should maybe add a comment that although these sec councils are being for-looped so the contract can be bricked by filling this array in practice the only entitity that can call this func is the dao, and they already have the power to upgrade this arbitrarily.

Either that or we add some hard coded max - eg 500

Reply:
i think max + a comment explaining makes sense. could be nice to gas benchmark to come up w a reasonable number